action.yml

```diff
@@ -35,3 +35,6 @@ inputs:
   githubBaseUrl:
     description: "If you use Github Enterprise, you can set this to the URL of your server (e.g. https://github.myorg.com/api/v3)"
     required: false
+  ignoreLabels:
```

Comment:
@amannn By the way, this option name looks good to you? If there is a more appropriate name, I will rename it.

Reply:
I think your proposed name is good!